tests/Feature/QuestionApiControllerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Quiz;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class QuestionApiControllerTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    public function test_create_question_success(): void
+    {
+        $quiz = Quiz::factory()->create();
+
+        $testData = [
+            'question' => 'What is the square root of 144',
+            'points' => '2',
+            'hint' => 'This is a hint',
+            'quiz_id' => $quiz->id,
+        ];
+
+        $response = $this->postJson('/api/questions', $testData);
+        $response->assertStatus(201);
+
+        $this->assertDatabaseHas('questions', $testData);
+    }
+
+    public function test_create_question_missing_data(): void
+    {
+        $response = $this->postJson('/api/questions', []);
+        $response->assertStatus(422);
+        $response->assertJsonValidationErrors(['question', 'points', 'quiz_id']);
+    }
+
+    public function test_create_question_invalid_data(): void
```

Comment:
Once you've added the exists check to quiz_id in the validation, I think it would be worth specifically testing for it. 

Send all valid data apart from the quiz_id, and make sure the valdation catches it